src/ui/settings.ts

```diff
@@ -60,6 +60,7 @@ export class SettingTab extends PluginSettingTab {
     }
 
     this.createTabAndContent('Custom', navEl, settingsEl, (el: HTMLElement, tabName: string) => this.generateCustomCommandSettings(tabName, el));
+    this.createTabAndContent('Custom', navEl, settingsEl, (el: HTMLElement, tabName: string) => this.generateCustomRegexReplacementSettings(tabName, el));
```

Comment:
This change here will cause problems. Try something more like:
```suggestion
    this.createTabAndContent('Custom', navEl, settingsEl, (el: HTMLElement, tabName: string) => {
      let tempContainer = el.createDiv();
      this.generateCustomCommandSettings(tabName, tempContainer);
      tempContainer = el.createDiv();
      this.generateCustomRegexReplacementSettings(tabName, tempContainer)
    });
```

Note that this is more of sudo code and will need to be verified to see if it works in its suggested state.